### PR TITLE
Doc/ledger

### PR DESCRIPTION
**Pull request by AureliaDolo**

### CODE REVIEW

**Review by yvan-sraka — DISMISSED** [review]

File: models/src/ledger.rs [hunk 1]
```diff
@@ -68,10 +68,13 @@ impl LedgerData {
     }
 }
 
+/// A balance change that can be applied to an address
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct LedgerChange {
+    /// Amount to add or substract
     pub balance_delta: Amount,
-    pub balance_increment: bool, // wether to increment or decrement balance of delta
+    /// wether to increment or decrement balance of delta
```
Comment (damip) [inline models/src/ledger.rs:76]: whether

**Review by damip — DISMISSED** [review]
LGTM with one typo

**Review by damip — APPROVED** [review]

**Comment by AureliaDolo** [comment]
bors merge

**Comment by damip** [comment]
bors merge

**Pull request merged**